gluecodium/src/main/resources/templates/ffi/FfiIsolateContextHeader.mustache

```diff
@@ -0,0 +1,59 @@
+{{!!
+  !
+  ! Copyright (C) 2016-2020 HERE Europe B.V.
+  !
+  ! Licensed under the Apache License, Version 2.0 (the "License");
+  ! you may not use this file except in compliance with the License.
+  ! You may obtain a copy of the License at
+  !
+  !     http://www.apache.org/licenses/LICENSE-2.0
+  !
+  ! Unless required by applicable law or agreed to in writing, software
+  ! distributed under the License is distributed on an "AS IS" BASIS,
+  ! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ! See the License for the specific language governing permissions and
+  ! limitations under the License.
+  !
+  ! SPDX-License-Identifier: Apache-2.0
+  ! License-Filename: LICENSE
+  !
+  !}}
+{{>ffi/FfiCopyrightHeader}}
+
+#pragma once
+
+#include <cstdint>
+
+{{#internalNamespace}}
+namespace {{.}}
+{
+{{/internalNamespace}}
+namespace ffi
+{
+class IsolateContext
+{
+private:
+    int32_t previous_id;
+    static thread_local int32_t current_id;
+
+public:
+    inline IsolateContext(int32_t isolate_id)
```

Comment:
explicit
And i would remove 'inline' all around class

Reply:
Done